routers/api/v1/api.go

```diff
@@ -461,6 +461,16 @@ func RegisterRoutes(m *macaron.Macaron) {
 					m.Combo("/:id").Get(repo.GetRelease).
 						Patch(reqToken(), reqRepoWriter(), bind(api.EditReleaseOption{}), repo.EditRelease).
 						Delete(reqToken(), reqRepoWriter(), repo.DeleteRelease)
+					m.Combo("/latest").Get(repo.GetLatestRelease)
+					m.Group("/:id", func() {
+						m.Combo("").Get(repo.GetRelease).
+							Patch(bind(api.EditReleaseOption{}), repo.EditRelease).
+							Delete(repo.DeleteRelease)
+						m.Group("/assets", func() {
+							m.Combo("").Get(repo.ListReleaseAttachments)
+							m.Combo("/:assetId").Get(repo.GetReleaseAttachment)
```

Comment:
This route is incorrect; see https://developer.github.com/v3/repos/releases/#get-a-single-release-asset